functions/Resolve-DbaNetworkName.ps1

```diff
@@ -1,90 +1,84 @@
-      This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
-      You should have received a copy of the GNU General Public License along with this program. If not, see http://www.gnu.org/licenses/.
+	  .NOTES
+	  	Tags: Network, Resolve
+		Original Author: Klaas Vandenberghe ( @PowerDBAKlaas )
 
-      .LINK
-      https://dbatools.io/Resolve-DbaNetworkName
+		Website: https://dbatools.io
+		Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+		License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
 
-      .EXAMPLE
-      Resolve-DbaNetworkName -ComputerName ServerA
+	  .LINK
+		https://dbatools.io/Resolve-DbaNetworkName
 
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for ServerA
-	
-      .EXAMPLE
-      Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress
+	  .EXAMPLE
+		Resolve-DbaNetworkName -ComputerName ServerA
 
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress
-	
-      .EXAMPLE
-      Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress, sql2014
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for ServerA
 
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress and sql2014
+	  .EXAMPLE
+		Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress
 
-      Get-SqlRegisteredServerName -SqlInstance sql2014 | Resolve-DbaNetworkName
-	
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for all SQL Servers returned by Get-SqlRegisteredServerName
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress
+
+	  .EXAMPLE
+		Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress, sql2014
+
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress and sql2014
+
+		Get-SqlRegisteredServerName -SqlInstance sql2014 | Resolve-DbaNetworkName
+
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for all SQL Servers returned by Get-SqlRegisteredServerName
   #>
 	[CmdletBinding()]
 	param (
 		[parameter(ValueFromPipeline)]
-		[Alias('cn', 'host', 'ServerInstance', 'Server', 'SqlServer')]
-		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
-    	[PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential,
+		[Alias('cn', 'host', 'ServerInstance', 'Server', 'SqlInstance')]
+		[string[]]$ComputerName = $env:COMPUTERNAME,
```

Comment:
So why does it break when you pass in a computer name that does not exist on the network?